packages/hash/external-services/docker-compose.yml

```diff
@@ -95,7 +95,8 @@ services:
       HASH_GRAPH_PG_PORT: "5432"
       HASH_GRAPH_PG_DATABASE: "${HASH_GRAPH_PG_DATABASE}"
       HASH_GRAPH_LOG_FORMAT: "${HASH_GRAPH_LOG_FORMAT:-pretty}"
-      RUST_LOG: "${HASH_GRAPH_LOG_LEVEL:-graph=trace,hash_graph=trace}"
+      RUST_LOG: "${HASH_GRAPH_LOG_LEVEL:-graph=trace,hash_graph=trace,tokio_postgres=debug}"
+      RUST_BACKTRACE: 1
```

Comment:
Yes, but I'd actually keep it because it greatly helped me debugging. `tokio_postgres=debug` implies, that queries are printed, which is pretty useful. `RUST_BACKTRACE` allows us to capture backtraces on errors, which is helpful in the case a query cannot be processed. It shouldn't happen often, but if it happens, it's nice to have it 🙂

Reply:
Ah makes sense 👍